pkg/query-service/app/traces/v4/query_builder.go

```diff
@@ -148,6 +148,59 @@ func BuildTracesFilterQuery(fs *v3.FilterSet) (string, error) {
 	return queryString, nil
 }
 
+func BuildTracesFilter(fs *v3.FilterSet) (string, error) {
```

Comment:
@shivanshuraj1333 why is this new function created? Any limitations with `BuildTracesFilterQuery`?

Reply:
added a todo: remove